.github/workflows/approve-snapshots.yml

```diff
@@ -0,0 +1,50 @@
+name: Approve snapshots
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  dispatch:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && github.event.comment.body == '/approve'
```

Comment:
I don't use the slash-command action any more, I just do a manual check here to decide whether to run the workflow or not. This workflow no longer triggers another workflow -- everything is self-contained.